Automattic/sensei
# Translate block ids during importing

### Changes proposed in this Pull Request

* Translates the ids of the outline block to the newly created ones during importing.
* It uses the existing methods to translate ids for lessons.
* For modules the matching is done by name, similarily with the rest of the import functionality. This doesn't handle very well cases where modules with the same name exist but there is no alternative with the current import/export file structure.
* Module description is not imported/exported as there is no column for it. @donnapep maybe we should look into that?
 
### Testing instructions

* Create courses and lessons with the outline block and try to cover many edge cases.
* Export the content.
* On another site or with the original content deleted imported the previously exported content.
* Observe that the courses and lessons are linked correctly in the outline block.
* Update a course which has an outline block by using import.
* Observe that the course is updated and no warnings are displayed.
